go.mod

```diff
@@ -2,10 +2,14 @@ module terraform-provider-delphix
 
 go 1.17
 
-require github.com/delphix/dct-sdk-go v1.0.0-beta6
+require (
+	github.com/delphix/dct-sdk-go v1.0.0-beta9
```

Comment:
Done. For the reasons mentioned here (https://go.dev/blog/v2-go-modules), I have tagged Go SDK Version as 1.2.0.

@arunskurian I think we had briefly discussed about this before and decided to go with 1.2.0 because there is really no DCT v1 Go SDK that is generated and maintained by the team.